qiita_db/metadata_template.py

```diff
@@ -579,6 +579,40 @@ def _check_template_class(self, md_template):
         if not isinstance(md_template, SampleTemplate):
             raise IncompetentQiitaDeveloperError()
 
+    def __setitem__(self, column, value):
+        r"""Sets the metadata value for the category `column`
+
+        Parameters
+        ----------
+        column : str
+            The column to update
+        value : str
+            The value to set. This is expected to be a str on the assumption
+            that psycopg2 will cast as necessary when updating.
+
+        Raises
+        ------
+        QiitaDBColumnError
+            If the column does not exist in the table
+        """
+        conn_handler = SQLConnectionHandler()
+
+        exists = conn_handler.execute_fetchone("""
+            SELECT column_name
+            FROM information_schema.columns
+            WHERE table_name='{0}'
+                AND column_name='{1}'""".format(self._dynamic_table, column))
```

Comment:
As unlikely as it might be for a (table, column) to exist in a schema other than `qiita`, I feel that this query should also verify that the table_catalog = qiita and the table_schema = qiita